index.html

```diff
@@ -154,6 +154,11 @@
             function loadBoard() {
                 clearBoard();
                 startSpinner();
+                doGET("/rest/product/" + bzProduct, function(response) {
+                    if (!(response.products[0].has_unconfirmed)) {
+                        document.getElementsByClassName("board-column").UNCONFIRMED.hidden = true;
+                    }
+                });
```

Comment:
I feel like there may be a race condition here? Maybe we should do this after the board columns get added below.